kiota_http/httpx_request_adapter.py

```diff
@@ -127,11 +127,10 @@ def start_tracing_span(self, request_info: RequestInformation, method: str) -> t
         Returns:
             The parent span.
         """
-        characters_to_decode = ParametersNameDecodingHandlerOption().characters_to_decode
-        uri_template = ParametersNameDecodingHandler.decode_uri_encoded_string(
-            request_info.url_template, characters_to_decode
-        )
-        parent_span_name = f"{method} - {uri_template}"
+        uri_template = (request_info.url_template if request_info.url_template else "UNKNOWN")
+        decoded_uri_template = unquote(uri_template)
```

Comment:
I don't believe so, the URI template syntax will probably fail proper decoding in some cases by the code we have in the handler (it's not a valid URL until the template is expanded, which means it might/will fail to parse as a URL)

Reply:
Oh ok. Missed that we're using the template